src/store/modules/order/actions.ts

```diff
@@ -175,9 +177,16 @@ const actions: ActionTree<OrderState, RootState> = {
         current.poHistory.items = poHistory;
         commit(types.ORDER_CURRENT_UPDATED, current);
         return poHistory;
-      } 
+      } else {
+        const current = state.current as any
```

Comment:
We can store the current variable before try catch block and then use it everywhere.

Reply:
Made the requested change.